verifiers/internal/gha/builder_test.go

```diff
@@ -77,7 +85,7 @@ func Test_VerifyBuilderIdentity(t *testing.T) {
 			workflow: &WorkflowIdentity{
```

Comment:
to go even further, we could define the entire structure and use that everywhere, and replace only the pats we need. That makes the code even more readable. See for example https://github.com/laurentsimon/slsa-policy/blob/main/pkg/release/internal/project/project_test.go#L497. Not a blocker for this PR

Reply:
I looked into that, but it seems a bit too complex for now. Instead i just made a new `builderGoSlsa3GitURL  = httpsGithubCom + trustedBuilderRepository + builderGoSlsa3`

Reply:
works for me.